app/static/js/map.js

```diff
@@ -263,25 +271,40 @@ const displayResult = function (map, pixel) {
     }
 };
 
-export function initializeMaps() {
+const $mapContainer = $(".map__container");
+let $arcticScreenReaderList = $("#map__screenreader--arctic");
+let $antarcticScreenReaderList = $("#map__screenreader--antarctic");
+
+
+export function initializeMaps(lazy=false) {
+    $arcticScreenReaderList = $("#map__screenreader--arctic");
+    $antarcticScreenReaderList = $("#map__screenreader--antarctic");
+
+    $arcticScreenReaderList.empty();
+    $antarcticScreenReaderList.empty();
+
     arcticResultsSource.clear(true);
     antarcticResultsSource.clear(true);
 
-    $(".map__container").removeClass("hidden");
+    $mapContainer.removeClass("hidden");
 
-    if (!arcticMap) {
+
+    if (!lazy || !arcticMap) {
         arcticMap = new Map({
             target: "map--arctic",
             view: arcticView,
             layers: [arcticLayer, arcticResultsLayer],
+            controls: defaultControls({attribution: false}).extend(
```

Comment:
The map attribution (it is a little 'i' icon in the lower right) was collapsed by default on one map and not the other and it looked really funny. This collapses them both by default.